src/gt4py/next/program_processors/runners/dace_fieldview/utility.py

```diff
@@ -106,3 +107,40 @@ def get_tuple_type(data: tuple[Any, ...]) -> ts.TupleType:
     return ts.TupleType(
         types=[get_tuple_type(d) if isinstance(d, tuple) else d.data_type for d in data]
     )
+
+
+def patch_gtir(ir: gtir.Program) -> gtir.Program:
```

Comment:
The description of this function is a bit shallow, I would increase it, probably add why the IR is patched.

Reply:
Agreed.